app/Http/Controllers/RequestWithdrawalController.php

```
@@ -0,0 +1,156 @@
+                        'uom_id'                => $item['uom_id'],
+                        'purpose_of_withdrawal' => $item['purpose_of_withdrawal'] ?? null,
+                        'metadata'              => $item['metadata'] ?? null,
+                        'created_at'            => now(),
+                        'updated_at'            => now(),
+                    ];
+                });
+
+                RequestWithdrawalItem::insert($items->toArray());
+
+                return $withdrawal->fresh(['warehouse', 'chargeable', 'items.item', 'items.uom']);
+            });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'store'. (Clean Code Rules)

(StaticAccess)